examples/tls/Cargo.toml

```diff
@@ -7,6 +7,7 @@ publish = false
 
 [dependencies]
 rocket = { path = "../../core/lib", features = ["tls", "mtls", "secrets", "http3-preview"] }
+tracing = "*"
```

Comment:
We definitely don't want to do this. I fixed this upstream already - check it out.